internal/server/place/golden/get_locations_ranking/country.json

```diff
@@ -5,1233 +5,1078 @@
-          {
-            "rank": 211,
-            "value": 85032,
-            "placeDcid": "country/IMN"
-          },
-          {
-            "rank": 212,
-            "value": 77265,
-            "placeDcid": "country/AND"
-          },
-          {
-            "rank": 213,
-            "value": 71991,
-            "placeDcid": "country/DMA"
-          },
-          {
-            "rank": 214,
-            "value": 65720,
-            "placeDcid": "country/CYM"
-          },
-          {
-            "rank": 215,
-            "value": 63903,
-            "placeDcid": "country/BMU"
-          },
-          {
-            "rank": 216,
-            "value": 63026,
-            "placeDcid": "country/GGY"
-          },
-          {
-            "rank": 217,
-            "value": 59194,
-            "placeDcid": "country/MHL"
-          },
-          {
-            "rank": 218,
-            "value": 57557,
-            "placeDcid": "country/MNP"
-          },
-          {
-            "rank": 219,
-            "value": 56367,
-            "placeDcid": "country/GRL"
-          },
-          {
-            "rank": 220,
-            "value": 55197,
-            "placeDcid": "country/ASM"
-          },
-          {
-            "rank": 221,
-            "value": 53192,
-            "placeDcid": "country/KNA"
-          },
-          {
-            "rank": 222,
-            "value": 48865,
-            "placeDcid": "country/FRO"
-          },
-          {
-            "rank": 223,
-            "value": 40812,
-            "placeDcid": "country/SXM"
-          },
-          {
-            "rank": 224,
-            "value": 39244,
-            "placeDcid": "country/MCO"
-          },
-          {
-            "rank": 225,
-            "value": 38718,
+            "value": 39226,
             "placeDcid": "country/TCA"
           },
           {
-            "rank": 226,
-            "value": 38659,
-            "placeDcid": "country/MAF"
-          },
-          {
-            "rank": 227,
-            "value": 38137,
+            "rank": 209,
+            "value": 38254,
             "placeDcid": "country/LIE"
           },
           {
-            "rank": 228,
-            "value": 33938,
+            "rank": 210,
+            "value": 34010,
             "placeDcid": "country/SMR"
           },
           {
-            "rank": 229,
+            "rank": 211,
             "value": 33691,
             "placeDcid": "country/GIB"
           },
           {
-            "rank": 230,
-            "value": 30237,
+            "rank": 212,
+            "value": 30423,
             "placeDcid": "country/VGB"
           },
           {
-            "rank": 231,
-            "value": 25987,
-            "placeDcid": "country/BES"
-          },
-          {
-            "rank": 232,
-            "value": 18092,
+            "rank": 213,
+            "value": 18174,
             "placeDcid": "country/PLW"
           },
           {
-            "rank": 233,
```

Comment:
@chejennifer also reported this... something to look into

Reply:
it looks like a bunch of countries got removed from the rankings .. is that also expected?

Reply:
Good point. It appears that the ones that got deleted are not actual "countries" but various territories. I have checked and none of the removed ones appears in the WDI raw data import. I will check to see if they were ever in the older import

Reply:
Ok spot checking with our last WDI import (raw csv), I don't see these territories in that file either. So this is def not caused by any data missing in the WDI refresh process.

Reply:
This issue is now solved thanks to @chejennifer's fix from https://github.com/datacommonsorg/mixer/pull/953.